projects/dsp-ui/src/lib/search/advanced-search/select-property/specify-property-value/search-decimal-value/search-decimal-value.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Inject, Input, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Constants } from '@dasch-swiss/dsp-js';
+import { Value, ValueLiteral } from '../operator';
+
+// https://stackoverflow.com/questions/45661010/dynamic-nested-reactive-form-expressionchangedafterithasbeencheckederror
+const resolvedPromise = Promise.resolve(null);
+
+@Component({
+  selector: 'dsp-search-decimal-value',
+  templateUrl: './search-decimal-value.component.html',
+  styleUrls: ['./search-decimal-value.component.scss']
+})
+export class SearchDecimalValueComponent implements OnInit {
+
+    // parent FormGroup
+    @Input() formGroup: FormGroup;
+
+    type = Constants.DecimalValue;
+
+    form: FormGroup;
+
+    constructor(@Inject(FormBuilder) private fb: FormBuilder) {
+    }
+
+    ngOnInit() {
+
+        this.form = this.fb.group({
+            decimalValue: [null, Validators.compose([Validators.required])]
+        });
+
+        resolvedPromise.then(() => {
+            // add form to the parent form group
+            this.formGroup.addControl('propValue', this.form);
+        });
+
+    }
+
+    ngOnDestroy() {
```

Comment:
Lifecycle interface OnDestroy should be implemented for method ngOnDestroy.

Reply:
Good catch! Fixed in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/107/commits/568f53e3a8fc2c89fa73ec7b366dc6904263435b